cmd/create.go

```diff
@@ -40,11 +40,14 @@ A full list of component types that can be deployed is available using: 'odo com
   # Create new Wildfly component with binary named sample.war in './downloads' directory
   odo create wildfly wildly --binary ./downloads/sample.war
 
-  # Create a Ruby component
-  odo create ruby
-	
-  # Create a Python component
-  odo create python
+  # List of ready-to-use examples
+  odo create httpd --git https://github.com/openshift/httpd-ex.git
```

Comment:
Does it really make sense to list all the possibilities here? 
It looks really polluted. And to be honest is there any benefit of it?

I like the `docs/examples.md` document. But I'm not sure about listing everything here.

Examples here should be there just for users to help them understand the command variations and structure

Reply:
We could include maybe one and a link to more examples? What do you think

Reply:
something like, `For more examples....`

Reply:
> We could include maybe one and a link to more examples? What do you think

That would be much better.